pkgs/modules/bun/default.nix

```diff
@@ -6,6 +6,10 @@ let
   extensions = [ ".json" ".js" ".jsx" ".ts" ".tsx" ];
 
   community-version = lib.versions.majorMinor bun.version;
+
+  package-json-runner = pkgs.callPackage ../../package-json-runner {
+    js-runtime = "bun";
```

Comment:
Why not use the absolute path to the executable?

Reply:
to get the nice name in the runner's derivation name 😅 wasn't sure if i'd be bringing it back or not - it's coming back in one of the next commits :)